src/services/spatiallm/__tests__/WorkspaceProcessor.test.ts

```
@@ -0,0 +1,135 @@
+
+      // Create mock image data
+      const imageData = new ImageData(100, 100);
+      for (let i = 0; i < imageData.data.length; i += 4) {
+        imageData.data[i] = 255;     // R
+        imageData.data[i + 1] = 0;   // G
+        imageData.data[i + 2] = 0;   // B
+        imageData.data[i + 3] = 255; // A
+      }
+
+      // Save the image data
+      await (processor as any).saveImageDataToPNG(imageData, outputPath);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)